sources/packages/api/src/services/student/student.service.ts

```diff
@@ -164,7 +164,7 @@ export class StudentService extends RecordDataModelService<Student> {
       phone: studentInfo.phone,
     };
     student.user = user;
-
+    student.sinConsent = true;
```

Comment:
same here.

Reply:
This is used by both ministry user and student. Considering the ministry user in particular, if the consent is not checked, the record cannot be saved.